build/Targets/VSL.Versions.targets

```diff
@@ -17,6 +17,18 @@
     <BuildNumberBuildOfTheDayPadded Condition="('$(BuildNumber)' != '') AND ($(BuildNumber.Split('.').Length) == 2)">$(BuildNumber.Split('.')[1].PadLeft(2,'0'))</BuildNumberBuildOfTheDayPadded>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
Regardless of whether it is a *.props or *.targets file, the point I was trying to make was that having a standardized way to expose this information, across all of our repos is very useful.

Inlining it here means that anyone making changes has to remember that it is there in Roslyn and here in ProjectSystem, and possibly somewhere else in another repo.

Where-as having it in `Dependencies.props` or `Dependencies.targets` files means that anyone looking to update the referenced nuget packages can go and check this one location on any repo.